doc/modules/preprocessing.rst

```diff
@@ -426,3 +426,41 @@ values than observed values.
 
 :class:`Imputer` can be used in a Pipeline as a way to build a composite
 estimator that supports imputation. See :ref:`example_missing_values.py`
+
+.. _polynomial_features:
+
+Generating polynomial features
+==============================
+
+Often it's useful to add complexity to the model by considering nonlinearity within input data. One basic representation is to use polynomial to get features' high-order and interaction terms, which is implemented in :class:`PolynomialFeatures`::
```

Comment:
Maybe say "consider nonlinear features of the input data. A simple and common method to use are polynomial features .... ".
Maybe also add a sentence linking to SVC with poly kernel.